From 9817b5717b305f39ecc3c192781d376576bbd0d1 Mon Sep 17 00:00:00 2001
From: Sveinung Kvilhaugsvik <sveinung84@users.sourceforge.net>
Date: Fri, 12 Feb 2016 15:23:33 +0100
Subject: [PATCH 3/4] Let ALLOW_CTRL set metamessage setting.

A connection with ALLOW_CTRL access level can set the meta server user
message via the metamessage command. It should therefore be allowed to set
it as a setting too.

A user that is able to set the meta server user message should be allowed to
view its current value too.
---
 server/settings.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/server/settings.c b/server/settings.c
index 3b2b5a0..c03b91f 100644
--- a/server/settings.c
+++ b/server/settings.c
@@ -2826,7 +2826,7 @@ static struct setting settings[] = {
           GAME_MIN_KICK_TIME, GAME_MAX_KICK_TIME, GAME_DEFAULT_KICK_TIME)
 
   GEN_STRING("metamessage", game.server.meta_info.user_message,
-             SSET_META, SSET_INTERNAL, SSET_RARE, SSET_SERVER_ONLY,
+             SSET_META, SSET_INTERNAL, SSET_RARE, SSET_TO_CLIENT,
              N_("Metaserver info line"),
              N_("User defined metaserver info line. For most of the time "
                 "a user defined metamessage will be used instead of an "
-- 
2.7.0

